packages/next/src/index.ts

```diff
@@ -35,12 +35,18 @@ export default class LogtoClient {
   private createNodeClient(request: NextApiRequest) {
     this.storage = new NextStorage(request);
 
-    return new NodeClient(this.config, {
-      storage: this.storage,
-      navigate: (url) => {
-        this.navigateUrl = url;
+    return new NodeClient(
+      {
+        ...this.config,
+        persistAccessToken: true,
```

Comment:
Should we leave this to user decision? If we do this, even if the user specificly set it to false in his app config, we will not respect that and still force use `true` in our SDK

Reply:
Oops, fixed.
Changed to default true, and the user can specificly set it to false